_data/negotiation-mastery-program.yaml

```diff
@@ -0,0 +1,174 @@
+title: Negotiation Mastery Program
+mission: Awaken the negotiator within—calm, clear, and creative.
+principles:
+  - 1st: Relax completely
+  - 2nd: Feel into emotional waves
+  - 3rd: Release tension inviting clarity
+  - 4th: Form reflection into reflex
+community:
+  - url: https://dojo.center/community
+practices:
+  - reflection:

```

Comment:
reflection section is good ... but I want to talk through the distinction between the continuous reflection and belt reflection.

I sense there is some confusion here (not in this file), but when it comes up in conversation